lexibank_bodtkhobwa.py

```diff
@@ -124,15 +124,16 @@ def cmd_install(self, **kw):
                 "format": "([\\S]+)( [\\S]+)*"
             })
             for idx in tqdm(data, desc='cldf the data'):
-                segments = ' '.join([mapper.get(x, x) for x in data[idx, 'tokens']])
-                morphemes = segments.split(' + ')
+                segments = [mapper.get(x, x) for x in data[idx, 'tokens']]
+                morphemes = ' '.join(segments).split(' + ')
+
                 concept = concepts.get(data[idx, 'concept'], '')
                 for lex in ds.add_lexemes(
                     Language_ID=data[idx, 'doculect'].lower(),
                     Parameter_ID=concept,
                     Form=data[idx, 'form'],
                     Value=data[idx, 'value'],
-                    Segments=morphemes,
+                    Segments=segments,
```

Comment:
This should be as in the original.

Reply:
I mean, I am not sure why this is actually working, but it must have to do with modifications by @xrotwang ? Anyway, checking the code here is probably important, the code in lexibank, I mean.

Reply:
Yes, I quickly checked before leaving. But as for my code working, note that I also changed lines 127 and 128, so that at this point `segments` is not a string (as in your original), but a list as expected.

Still, now that we are moving to the developing version (or HEAD, to be precise), no need to worry about this. I will check everything again tomorrow.